packages/core/src/schema/models/runErrors.ts

```diff
@@ -19,26 +18,23 @@ export const runErrorEntities = latitudeSchema.enum('run_error_entity_enum', [
   ErrorableEntity.EvaluationResult,
 ])
 
+type RunErrorDetails<C extends RunErrorCodes> =
+  C extends RunErrorCodes.ChainCompileError
+    ? { compileCode: string; message: string }
+    : never
+
 export const runErrors = latitudeSchema.table(
   'run_errors',
   {
     id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
     code: errorCodeEnum('code').notNull(),
-    errorableType: runErrorEntities('errorable_type'),
-    errorableId: bigint('errorable_id', { mode: 'number' }),
-    providerLogId: bigint('provider_log_id', { mode: 'number' }).references(
```

Comment:
I decided that I'd add this pointer in the other direction. A provider log would have a nullable error